internal/notifier/factory.go

```diff
@@ -89,6 +89,8 @@ func (f Factory) Notifier(provider string) (Interface, error) {
 		n, err = NewOpsgenie(f.URL, f.ProxyURL, f.CertPool, f.Token)
 	case v1beta1.AlertManagerProvider:
 		n, err = NewAlertmanager(f.URL, f.ProxyURL, f.CertPool)
+	case v1beta1.GrafanaProvider:
+		n, err = NewGrafana(f.URL, f.ProxyURL, f.Token)
```

Comment:
Any reason for not supporting self-signed certs like all the other providers?

Reply:
I've submitted pushed another commit fixing this